### DISCUSSION TITLE
DRAW.IO (old version) x DIAGRAMS.NET

### DISCUSSION THREAD

**Discussion by andrecwb86** [jgraph/drawio-desktop#722] [General]
I need to install Draw.IO (version 7.9.5) But whenever I install it automatically opens version 15.3.5. Is there any way where I can unlink this update? Basically I have already removed all the extensions previously installed in the operating system. (C:\Program Files\draw.io) and some records in regedit .

**Comment by davidjgraph** [comment]
That version has critical security holes, why do you need that version specifically?

**Reply by andrecwb86** [reply]
because in this version I use it doesn't truncate the text of the <text> tag that is inside the .svg in my flowcharts. But this can be a private thing here with me.

**Reply by andrecwb86** [reply]
no matter which version I'm using, I need to unlink the version that automatically upgrades to Diagrams.NET 15.3.6.

**Comment by davidjgraph** [comment]
You're using a version that acts as a browser. It's loading the web site within the app, so you'll see whatever is currently online. Have a look in the desktop project releases list.
